crates/fluvio-controlplane-metadata/Cargo.toml

```diff
@@ -36,6 +36,7 @@ fluvio-types = { version = "0.3.1", path = "../fluvio-types" }
 fluvio-stream-model = { path = "../fluvio-stream-model", version = "0.8.0" }
 fluvio-protocol = { path = "../fluvio-protocol", version = "0.8", features = [
     "record",
+    "use_serde",
```

Comment:
we still want to make `serde` optional

Reply:
Yeah with [630aa06](https://github.com/infinyon/fluvio/pull/2738/commits/630aa06d16fae114d94efae22924cd9060650394) this is back to the current state.